### ISSUE TITLE
Should multiple annotations for an item have line breaks in between?

### ISSUE THREAD

**Issue by bimimicah** [microsoft/XmlNotepad#216]
Please show multiple &lt;documentation&gt; elements in the schema on separate lines.

Example from VS Code w/ Redhat's XML extension:
![image](https://user-images.githubusercontent.com/34111930/184441692-149d90e8-ba15-4fea-98fa-a169022b5456.png)

Currently in XML Notepad:
![image](https://user-images.githubusercontent.com/34111930/184442008-7f8bcea1-538f-4223-8e34-e69b91366a56.png)

```
<xs:enumeration value="value1">
	<xs:annotation>
		<xs:documentation>Documentation for attribute value1</xs:documentation>
		<xs:documentation>Documentation line 2 for attribute value1</xs:documentation>
		<xs:documentation>Documentation line 3 for attribute value1</xs:documentation>
	</xs:annotation>
</xs:enumeration>
```

**Comment by lovettchris** [comment]
great bug, thanks!

**Comment by lovettchris** [comment]
Fixed in 2.8.0.62
